link/kprobe_test.go

```diff
@@ -307,19 +307,6 @@ func TestKprobeTraceFSGroup(t *testing.T) {
 	c.Assert(err, qt.Not(qt.IsNil))
 }
 
-func TestDetermineRetprobeBit(t *testing.T) {
```

Comment:
I removed this because it tests something that might change in the future (if I understand correctly).

Reply:
Hey, could you elaborate?

Reply:
My thinking was:  Asserting that the value is 0 hardcodes an assumption the kernel wants to be able to change. Checking that err == nil just tests uint64FromFile, which is covered indirectly via kprobe, etc. tests.

We can add an explicit test of uint64FromFile if you want.

Reply:
> Checking that err == nil just tests uint64FromFile, which is covered indirectly via kprobe, etc. tests.

Realized that too late, thanks!